### PR TITLE
CEDS-3244 Upgrade to Play 2.8

### PR DESCRIPTION
**Pull request by udinobi**

### CODE REVIEW

File: test/connectors/CodeLinkConnectorISpec.scala [hunk 1]
```diff
@@ -19,7 +19,7 @@ package connectors
 import config.AppConfig
 import org.mockito.Mockito.{reset, when}
 
-class CodeLinkConnectorSpec extends ConnectorSpec {
+class CodeLinkConnectorISpec extends ConnectorISpec {
```
Comment (glidester) [inline test/connectors/CodeLinkConnectorISpec.scala:22]: Why rename to ISpec? This isn't really an integration test, its part of the unit tests.

Comment (udinobi) [reply test/connectors/CodeLinkConnectorISpec.scala:22]: But it extends ConnectorISpec which in turn extends `GuiceOneAppPerSuite` and `WiremockTestServer` which are used for ITs

Comment (glidester) [inline test/connectors/CodeLinkConnectorISpec.scala:22]: This is not really an integration test (no use of wiremock) so I'd leave out the "ISpec" postfix.

File: test/connectors/CodeListConnectorISpec.scala [hunk 2]
```diff
@@ -23,7 +23,7 @@ import java.util.Locale.{ENGLISH, JAPANESE}
 
 import scala.collection.immutable.ListMap
 
-class CodeListConnectorSpec extends ConnectorSpec {
+class CodeListConnectorISpec extends ConnectorISpec {
```
Comment (glidester) [inline test/connectors/CodeListConnectorISpec.scala:26]: This is not really an integration test (no use of wiremock) so I'd leave out the "ISpec" postfix.

File: test/connectors/exchange/ExportsDeclarationExchangeSpec.scala [hunk 3]
```diff
@@ -23,12 +23,12 @@ import forms.declaration.additionaldeclarationtype.AdditionalDeclarationType.Add
...
 import models.declaration._
 import models.{DeclarationStatus, DeclarationType, ExportsDeclaration}
-import org.scalatest.{Matchers, WordSpec}
+import base.UnitSpec
 import org.scalatestplus.mockito.MockitoSugar
 import play.api.libs.json.{JsSuccess, Json}
 import services.cache.ExportsDeclarationBuilder
 
-class ExportsDeclarationExchangeSpec extends WordSpec with Matchers with ExportsDeclarationBuilder with MockitoSugar {
+class ExportsDeclarationExchangeSpec extends UnitSpec with ExportsDeclarationBuilder with MockitoSugar {
```
Comment (glidester) [inline test/connectors/exchange/ExportsDeclarationExchangeSpec.scala:31]: Shouldn't this also be UnitWithMocksSpec

Comment (udinobi) [reply test/connectors/exchange/ExportsDeclarationExchangeSpec.scala:31]: Good spot :+1:

File: test/connectors/CodeLinkConnectorISpec.scala (hunk 1 above)
Comment (udinobi) [reply test/connectors/CodeLinkConnectorISpec.scala]: I'll change then the extended trait.

File: test/connectors/CodeListConnectorISpec.scala (hunk 2 above)
Comment (udinobi) [reply test/connectors/CodeListConnectorISpec.scala]: I'll change then the extended trait.

**Review by glidester — APPROVED** [review]

**Pull request merged**